pom.xml

```diff
@@ -412,6 +411,15 @@
                 <artifactId>kotlin-stdlib</artifactId>
                 <version>1.6.10</version>
             </dependency>
+            <!-- Transitive dependency of kubernetes-client-java in kubernetes-extension,
+            schema-repo in avro-extensions, and com.google.caliper:caliper in druid-server
```

Comment:
also druid-ranger-security?

Reply:
ha, yes. I have it in the PR description but not in the comment.